src/lib/accessibility/Behaviors/Popup/PopupBehavior.ts

```diff
@@ -0,0 +1,34 @@
+import { Accessibility } from '../../interfaces'
+import * as keyboardKey from 'keyboard-key'
+
+const isButton = props => {
+  return props && props.trigger && props.trigger.props && props.trigger.props.as === 'button'
```

Comment:
also, what about links (`a` tags), other `Button` components rendered as something else, etc?

Reply:
I am wondering if the best way is inside the Popup behavior to be handling the trigger's accessibility. This can be any component, and if the component itself already has some accessibility defined, isn't that enough?